lib/card.rb

```diff
@@ -0,0 +1,18 @@
+class Card
+
+  attr_reader :id
+
+  def initialize(id, attributes)
+    @id = id
+    @attributes = attributes
+  end
+
+  def method_missing(method_name, *args)
```

Comment:
this is kind of opaque. uses of `method_missing` overrides can be really helpful, or just confusing and impossible to debug. Why'd you choose to use it here?